src/components/Button/MyButton.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Box, Button } from '@mui/material';
+import { getButtonStyles } from './Button.styles';
+
+const MyButton = ({ children, type, buttonStyles }) => {
```

Comment:
Is mybutton name sounds good?

Reply:
No it doesn't sound good. I'm updating it's name.